### DISCUSSION TITLE
VS Code, The icon is not displayed when the plug-in is installed

### DISCUSSION THREAD

**Discussion by Maogd** [community/community#14763] [Copilot Conversations]
Vscode does not display the icon after installing the Copilot plug-in, so it cannot be enabled, nor can it be uninstalled and reinstalled

**Comment by lucalves** [comment]
Hi @Maogd!

Please have a look at [11684#discussioncomment](https://github.com/github/feedback/discussions/11684#discussioncomment-2200377) it and see if it will fix it.
